File: android/src/main/java/com/github/quadflask/react/navermap/RNNaverMapViewTextureManager.java
package com.github.quadflask.react.navermap;

import androidx.annotation.NonNull;
import com.facebook.react.bridge.ReactApplicationContext;
import com.naver.maps.map.NaverMapOptions;

public class RNNaverMapViewTextureManager extends RNNaverMapViewManager {
    public RNNaverMapViewTextureManager(ReactApplicationContext context) {
        super(context);
    }

    @NonNull
    @Override
    public String getName() {
        return "RNNaverMapViewTexture";
    }

    @Override
    protected NaverMapOptions getNaverMapViewOptions() {
        return new NaverMapOptions()
                .useTextureView(true)
                .translucentTextureSurface(true);
    }
}

File: android/src/main/java/com/github/quadflask/react/navermap/RNNaverMapViewTextureManager.kt
package com.github.quadflask.react.navermap

import com.facebook.react.bridge.ReactApplicationContext
import com.naver.maps.map.NaverMapOptions

class RNNaverMapViewTextureManager(context: ReactApplicationContext) : RNNaverMapViewManager(context) {
  override fun getName() = "RNNaverMapViewTexture"

  override val naverMapViewOptions: NaverMapOptions
    get() = NaverMapOptions()
      .useTextureView(true)
      .translucentTextureSurface(true)
}
